content/docs/getting-started/connect-dbaas.md

```diff
@@ -64,3 +65,23 @@ Configure `cURL` with DBaaS credentials.
 ```sh
 alias curl="curl -s -k -u \"${NUODB_CP_USER}:${NUODB_CP_PASSWORD}\""
 ```
+
+### Setting up the Terraform Provider
+
+To use terraform to manage your databases, you will want to use the `nuodbaas` provider.
+In production, you will want to specify connection parameters explicitly, but, for prototyping, the provider can find its configuration in the environment varaibles set above.
+
+```terraform
+terraform {
+  required_providers {
+    nuodbaas = {
+      source  = "registry.terraform.io/nuodb/nuodbaas"
+      version = "1.1.0"

```

Comment:
The latest version is 1.2.0.

Reply:
Good catch. We need to update the terraform registry docs.